src/pkg/bundle/deploy.go

```diff
@@ -464,3 +479,34 @@ func setTemplatedVariables(templatedVariables string, pkgVars map[string]string)
 	})
 	return replacedValue
 }
+
+// verifyAndAddFileTo
+func (b *Bundle) verifyAndAddFileTo(helmFileVals []string, filePath string, key string) ([]string, error) {
+	verifiedPath, err := formAndCheckFilePath(b.cfg.DeployOpts.Config, filePath)
+	if err != nil {
+		return nil, err
+	}
+	helmVal := fmt.Sprintf("%s=%v", key, verifiedPath)
+	return append(helmFileVals, helmVal), nil
+}
+
+// formAndCheckFilePath merges relative paths together to form full path and checks if the file exists
+func formAndCheckFilePath(anchorPath string, filePath string) (string, error) {
```

Comment:
It's idiomatic in Go to have short variable and function names (where it makes sense!). In the case of `verifyAndAddFile` and `formAndCheckFilePath`, IMO it's unnecessary to say that we're "verifying" and "checking" in the function names. I think `formandCheckFilePath` could be `formFilePath` and `verifyAndAddFile` could be `addFileValue` or something like that